app/Http/Controllers/UsersController.php

```diff
@@ -131,6 +132,11 @@ public function show($id)
             ])
         );
 
-        return view('users.show', compact('user', 'userArray', 'achievements'));
+        // The default order of the profile sections,
+        // used when the particular user has no profile customization
+        // object created.
+        $defaultProfileOrder = UserProfileCustomization::$sections;
```

Comment:
just set the default in `UserProfileCustomization`'s `__construct`. Look how cover's default is set.